docs/source/technical_design/utilities/scripts/openKAT_install.sh

```diff
@@ -0,0 +1,285 @@
+if [[ ${U} == "" ]]; then
+	echo  "Create mula user with password ${MULADB_PASSWORD}..."
+	sudo -u postgres psql -c "CREATE USER mula WITH PASSWORD '${MULADB_PASSWORD}';"
+
+	echo "Grant database muladb to mula user..."
+	sudo -u postgres psql -c 'GRANT ALL ON SCHEMA public TO mula;' mula_db
+else
+	echo "Change password mula user to ${MULADB_PASSWORD}..."
+	sudo -u postgres psql -c "ALTER USER mula WITH PASSWORD '${MULADB_PASSWORD}';"
+fi
+
+echo Step 4.6 - Update configs
+echo "Step 4.6.1 - Update ROCKY_DB_PASSWORD in /etc/kat/rocky.conf to ${ROCKY_DB_PASSWORD}"
+sudo sed -i "/ROCKY_DB_PASSWORD=/s/.*/ROCKY_DB_PASSWORD=${ROCKY_DB_PASSWORD}/" /etc/kat/rocky.conf
+
+echo "Step 4.6.2 - Update BYTES_PASSWORD in /etc/kat/bytes.conf, /etc/kat/boefjes.conf, /etc/kat/mula.conf and /etc/kat/rocky.conf to ${BYTESDB_PASSWORD}"
+sudo sed -i "/BYTES_PASSWORD=/s/.*/BYTES_PASSWORD=${BYTESDB_PASSWORD}/" /etc/kat/bytes.conf
+sudo sed -i "/BYTES_PASSWORD=/s/.*/BYTES_PASSWORD=${BYTESDB_PASSWORD}/" /etc/kat/boefjes.conf
+sudo sed -i "/BYTES_PASSWORD=/s/.*/BYTES_PASSWORD=${BYTESDB_PASSWORD}/" /etc/kat/mula.conf
+sudo sed -i "/BYTES_PASSWORD=/s/.*/BYTES_PASSWORD=${BYTESDB_PASSWORD}/" /etc/kat/rocky.conf
+
+echo "Step 4.6.3 - Update BYTES_DB_URI in /etc/kat/bytes.conf to ${BYTESDB_PASSWORD}"
+sudo sed -i "/BYTES_DB_URI=/s/.*/BYTES_DB_URI=postgresql:\/\/bytes:${BYTESDB_PASSWORD}@localhost\/bytes_db/" /etc/kat/bytes.conf
+
+echo "Step 4.6.4 - Update KATALOGUS_DB_URI in /etc/kat/boefjes.conf to ${KATALOGUSDB_PASSWORD}"
+sudo sed -i "/KATALOGUS_DB_URI=/s/.*/KATALOGUS_DB_URI=postgresql:\/\/katalogus:${KATALOGUSDB_PASSWORD}@localhost\/katalogus_db/" /etc/kat/boefjes.conf
+
+echo "Step 4.6.5 - Update SCHEDULER_DSP_BROKER_URL in /etc/kat/mula.conf to ${RABBITMQ_PASSWORD}"
+sudo sed -i "/SCHEDULER_DSP_BROKER_URL=/s/.*/SCHEDULER_DSP_BROKER_URL=amqp:\/\/kat:${RABBITMQ_PASSWORD}@localhost:5672\/kat/" /etc/kat/mula.conf
+
+echo "Step 4.6.6 - Update SCHEDULER_RABBITMQ_DSN in /etc/kat/mula.conf to ${RABBITMQ_PASSWORD}"
+sudo sed -i "/SCHEDULER_RABBITMQ_DSN=/s/.*/SCHEDULER_RABBITMQ_DSN=amqp:\/\/kat:${RABBITMQ_PASSWORD}@localhost:5672\/kat/" /etc/kat/mula.conf
+
+echo "Step 4.6.7 - Update SCHEDULER_DB_DSN in /etc/kat/mula.conf to ${MULADB_PASSWORD}"
+sudo sed -i "/SCHEDULER_DB_DSN=/s/.*/SCHEDULER_DB_DSN=postgresql:\/\/mula:${MULADB_PASSWORD}@localhost\/mula_db/" /etc/kat/mula.conf
+
+echo "Step 4.6.8 - Update QUEUE_URI in rocky.conf, bytes.conf, boefjes.conf, octopoes.conf to ${RABBITMQ_PASSWORD}"
+sudo sed -i "/QUEUE_URI=/s/.*/QUEUE_URI=amqp:\/\/kat:${RABBITMQ_PASSWORD}@localhost:5672\/kat/" /etc/kat/rocky.conf
+sudo sed -i "/QUEUE_URI=/s/.*/QUEUE_URI=amqp:\/\/kat:${RABBITMQ_PASSWORD}@localhost:5672\/kat/" /etc/kat/bytes.conf
+sudo sed -i "/QUEUE_URI=/s/.*/QUEUE_URI=amqp:\/\/kat:${RABBITMQ_PASSWORD}@localhost:5672\/kat/" /etc/kat/boefjes.conf
+sudo sed -i "/QUEUE_URI=/s/.*/QUEUE_URI=amqp:\/\/kat:${RABBITMQ_PASSWORD}@localhost:5672\/kat/" /etc/kat/octopoes.conf
```

Comment:
All the RabbitMQ URLs should use `127.0.0.1` instead of `localhost`. If you specifiy `localhost` it will try to connect to IPv6 `::1` first and log an error about not being able to connect. This has already been changed in the installation manual.